docs/en/observability/monitor-gcp.asciidoc

```diff
@@ -0,0 +1,457 @@
+<3> Collects metrics within the `elastic-education` project-id.
+<4> Collects metrics every minute
+<5> The GCP credential file that you generated earlier. (Don't forget to create
+the file if it does not exist and to use the correct full path).
+
+. Now, test the input to check if {metricbeat} can collect data.
+[source,bash]
++
+----
+./metricbeat test modules gcp
+----
+
+. Input and output are ready. Next, start {metricbeat} to collect the data.
++
+[source,bash]
+----
+./metricbeat -e
+----
+
+. Finally, go to Kibana and open the "[{metricbeat} GCP] Compute Overview"
+dashboard.
++
+image:monitor-gcp-compute-overview-dashboard.png[{metricbeat} compute overview dashboard]
+
+[discrete]
+=== Step 4: Install and configure {filebeat}
+
+Now that {metricbeat} is up and running you will configure {filebeat} to
+collect Google Cloud logs.
+
+[discrete]
+==== Install {filebeat}
+
+On a new terminal window connected to the GCP instance, download and install
+{filebeat}.
+
+include::{beats-repo-dir}/tab-widgets/install-widget-filebeat.asciidoc[]
+
+[discrete]
+==== Set up assets
+
+Similar to {metricbeat}, {filebeat} comes with predefined assets for parsing,
+indexing, and visualizing your data.
+Run the following command to load these assets. It may take a few minutes.
+
+[source,bash]
+----
+./filebeat setup -e -E 'cloud.id=YOUR_DEPLOYMENT_CLOUD_ID' -E 'cloud.auth=elastic:YOUR_SUPER_SECRET_PASS'
+----
+
+[IMPORTANT]
+====
+Setting up {filebeat} is an admin-level task that requires extra privileges.
+As a best practice, https://www.elastic.co/guide/en/beats/filebeat/7.11/privileges-to-setup-beats.html[use an administrator role to set up]
+and a more restrictive role for event publishing (which you will do next).
+====
+
+[discrete]
+==== Configure {filebeat} output
+
+Next, you are going to configure {filebeat} output to {ess}.
+
+. Use the {filebeat} keystore to store {filebeat-ref}/keystore.html[secure
+settings]. Store the Cloud ID in the keystore.
++
+[source,bash]
+----
+./filebeat keystore create
+echo -n "<Your Deployment Cloud ID>" | ./filebeat keystore add CLOUD_ID --stdin
+----
+
+. To store logs in {es} with minimal permissions, create an API key to send
+data from {filebeat} to {ess}. Log into Kibana (you can do so from the Cloud
+Console without typing in any permissions) and select *Management* -> *Dev
+Tools*. Then, send the following request:
++
+[source,console]
+----
+POST /_security/api_key
+{
+  "name": "filebeat-monitor-gcp",
+  "role_descriptors": {
+    "filebeat_writer": {
+      "cluster": ["monitor", "read_ilm", "cluster:admin/ingest/pipeline/get", "cluster:admin/ingest/pipeline/put"],
+      "index": [
+        {
+          "names": ["filebeat-*"],
+          "privileges": ["view_index_metadata", "create_doc"]
+        }
+      ]
+    }
+  }
+}
+----
++
+[NOTE]
+----
+{filebeat} needs two special cluster permissions
+`cluster:admin/ingest/pipeline/get` and `cluster:admin/ingest/pipeline/put`.
```

Comment:
this note is hard to read and provides no context why it's here

![image](https://user-images.githubusercontent.com/667544/109948654-d436ad80-7cda-11eb-95d0-bcf2d1c85ff0.png)

Reply:
tried to improve readability and context. let me know what do you think.